content/docs/user-guide/packaging/project-export-ios.md

```diff
@@ -0,0 +1,84 @@
+
+{{< image-width "/images/user-guide/packaging/project-export-ios/atom-sampleviewer-configuration-page.png" "700">}}
+
+Then go to the signing and capabilities tab, and setup the provisioning profile to link to your AppleID, slightly customize the bundle identifier (add any string you want, for example: 'test'), and check "Automatically manage signing."
+
+{{< image-width "/images/user-guide/packaging/project-export-ios/signing-and-capabilities-page.png" "700">}}
+
+Now click on the bar in the region labeled “AtomSampleViewer.GameLauncher”, and scroll through the drop down to click “Edit Scheme.”
+
+{{< image-width "/images/user-guide/packaging/project-export-ios/edit-scheme.png" "700">}}
+
+Set the desired build configuration, and in the arguments section, set the relevant CLI parameters. An example configuration is shown below:
+
+{{< image-width "/images/user-guide/packaging/project-export-ios/scheme-configuration-debug-build.png" "700">}}
+{{< image-width "/images/user-guide/packaging/project-export-ios/scheme-configuration-cli-params.png" "700">}}
+
+Now make sure to connect your iOS device to the computer. Xcode should recognize your device, and it should be compatible with the project. If not, follow the XCode error prompts to trouble shoot the issue. Updating may be required. Once this is all correct, press the play button to build the project. This can take some time.
+
+Once everything is done, assuming you used the sample configuration, you should get the following on your iOS device:
+
+{{< image-width "/images/user-guide/packaging/project-export-ios/o3de-atom-sampleviewer-ios-device-result.png" "700">}}
+
+## iOS Export Script
+O3DE ships with an experimental [iOS Export Script](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py), capable of generating an Xcode project file to handle standard use cases of O3DE projects on iOS. This script is only designed to run from an O3DE source installation on a macOS machine.
+
+The export script has 2 primary sections: the function [`export_ios_xcode_project`](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L16) and the [startup code that only runs if the script is invoked by the CLI](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L73). In depth discussion on these two sections will be made in the [Developer Guide](https://docs.o3de.org/docs/engine-dev/) in the future.
+
+
+### Usage
+To use the export script, you can issue the arguments for this script at the same time that you are running the `export-project` command, so long as you are using this script as your designated export script. The arguments specific to the script will be deferred until the script begins running.
+
```

Comment:
Could you include a command-line sample?

Reply:
Yes, but it will just include dummy values for now